src/api/posts.ts

```diff
@@ -1,11 +1,25 @@
-import { getRequest } from './api';
+import { Post } from '../react-app-env';
 
-export const getAllPosts = async (): Promise<Post[]> => getRequest('/posts');
+export const BASE_URL = 'https://mate.academy/students-api/';
 
-export const getUserPosts = async (userId: number): Promise<Post[]> => (
-  getRequest(`/posts?userId=${userId}`)
-);
+export async function getPosts(userId: string): Promise<Post[]> {
```

Comment:
Why to change the name to `getPosts`? Previous was more descriptive and the same as described in requirements of task (`getUserPosts`)